0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,248 @@
+
+## Outgoing (Transfer, Ledger, Amount)
+The transfer/ledger/amount for which a connector plays the sender role. See also [Incoming](#incoming-transfer-ledger-amount).
+
+## Payment
+The movement of assets from one party to another. A payment consists of one or more [ledger transfers](#transfer). Roughly synonymous with
+[transaction](#transaction), and preferable in source code where 'transaction' may be confused with database transactions.
+The word payment has an emphasis on the context of the negotiation between the sender and receiver. For instance,
+it may be paying a debt, or a purchase, it may be a pull payment or a push payment. All these aspects which put a transaction in a wider
+context and link it to real-world interactions, contracts, and motivations between parties make a transaction into a payment. A
+[cyclic transaction](#cyclic-transaction) is generally not considered a payment, because the sending party of such a transaction is also the receiving party.
+
+## Payment channel
+A method used by two parties on the same ledger to make multiple transfers such that the ledger enforces the final net positions without the ledger being informed of each individual transfer. One or both parties will generally escrow some of their assets on the ledger and transfers are made by exchanging "claims" that update the net balance. When one of the parties wishes to close the channel, they submit the final balance to the ledger. Different implementations have different rules about which party is allowed to close the channel, the consequences of doing so, and the mechanisms for settling disputes. For more details see [Bitcoin Payment Channels](https://en.bitcoin.it/wiki/Payment_channels). Note that payment channels are similar to [trustlines](#trustlines) except the ledger is used to escrow one or both parties' assets and provide clear resolution mechanisms for disputes over the final net positions.
+
+A payment channel is similar to a trustline, but each party parks an asset on an "underlying ledger", equal to the other party's maximum balance.
+Apart from disputes canceling each other out, and out-of-band settlement, disputes can be settled by each party claiming the other party's parked
+asset. This way, two strangers can run a ledger with properties similar to a trustline, but without fully trusting each other.
+
+## Peer balance
+See [trustline](#trustline).
+
+## Peering
+Connectors "peer" with one another to exchange information used to determine the best route for a payment, and to route it (connect its chain of
+transfers during the [preparation phase](#preparation-phase) and [execution phase](#execution-phase).
+A peering relationship is usually implemented using either a [trustline](#trustline) or a [payment channel](#payment-channel)..
+
+## Preparation Phase
+The first phase of an ILP payment, in which the ILP packet is passed from the sender, via the connector(s), to the receiver.
```

Comment:
Also, money is put on hold. That is the most important thing to mention about this phase!